packages/synapse-react-client/src/components/GoalsV2/GoalsV2.tsx

```diff
@@ -0,0 +1,235 @@
+          files.requestedFiles
+            .filter(el => el.preSignedURL !== undefined)
+            .map(el => el.preSignedURL!),
+        )
+      } catch (e) {
+        console.error('Error on get data', e)
+        setError(e)
+      }
+    }
+    getData()
+  }, [entityId, accessToken, queryResultBundle])
+
+  const tableIdColumnIndex = getFieldIndex(
+    ExpectedColumns.TABLEID,
+    queryResultBundle,
+  )
+  const countSqlColumnIndex = getFieldIndex(
+    ExpectedColumns.COUNT_SQL,
+    queryResultBundle,
+  )
+
+  const titleColumnIndex = getFieldIndex(
+    ExpectedColumns.TITLE,
+    queryResultBundle,
+  )
+  const summaryColumnIndex = getFieldIndex(
+    ExpectedColumns.SUMMARY,
+    queryResultBundle,
+  )
+  const linkColumnIndex = getFieldIndex(ExpectedColumns.LINK, queryResultBundle)
+
+  const goalsDataArray: GoalsV2DataProps[] = []
+
+  queryResultBundle?.queryResult!.queryResults.rows.forEach((el, index) => {
+    const values = el.values as string[]
+    if (values.some(value => value === null)) {
+      console.warn('Row has null value(s) when no nulls expected')
+    }
+    const tableId =
+      tableIdColumnIndex > -1 ? values[tableIdColumnIndex] : undefined
+    let countSql
+    if (countSqlColumnIndex > -1 && values[countSqlColumnIndex]) {
+      countSql = values[countSqlColumnIndex]
+    } else if (tableId) {
+      countSql = `SELECT * FROM ${tableId}`
+    }
+    const title = values[titleColumnIndex]
+    const summary = values[summaryColumnIndex]
+    const link = values[linkColumnIndex]
+    const asset = assets?.[index] ?? ''
+    const goalsV2DataProps: GoalsV2DataProps = {
+      countSql,
+      title,
+      summary,
+      link,
+      asset,
+    }
+    goalsDataArray.push(goalsV2DataProps)
+  })
+
+  return (
+    <Box
+      sx={{
+        backgroundColor: 'rgba(0, 128, 0, 0.05)',
+        height: '560px',
+        padding: '80px',
+      }}
+    >
+      <Box
+        sx={{
+          textAlign: 'center',
+          paddingBottom: '30px',
+        }}
+      >
+        <Typography
+          variant="headline1"
+          style={{
+            color: 'grey.1000',
+            fontFamily: 'Merriweather',
+            fontSize: '32px',
+          }}
+          sx={{
+            pt: '30px',
+            mb: '40px',
+            mx: 'auto',
+            width: 'max-content',
+            borderTop: '3px solid rgba(128, 128, 128, 0.25)',
+          }}
+        >
+          What's in the Portal?
+        </Typography>
+        <Link
+          href={dataLink}
+          target="_blank"
+          rel="noopener noreferrer"
+          fontFamily="Lato"
+          fontSize="18"
+          fontStyle="semi-bold"
+        >
+          <Button
+            href={dataLink}
+            target="_blank"
+            rel="noopener noreferrer"
+            variant="contained"
+            sx={{
+              backgroundColor: '#5BA998',
+              border: '1px solid white',
+              boxShadow: 'none',
```

Comment:
You can just remove the box shadow from all states (hover, active) with `!important` here. Should be safe since it only applies to this specific button.

That said, I think @kianamcc is moving these button styles to the EL Home Page theme override in #1456, so you might be able to remove some of these style overrides entirely 

```suggestion
              boxShadow: 'none !important',
```